bin/opam-web.sh

```diff
@@ -22,11 +22,8 @@ mv opam-repository/* .
 mv opam-repository/.git .
 rm -rf opam-repository
 
-# Overwrite 'repo' file, and dispatch all non-standard versions
-cat <<EOF >repo
-opam-version: "2.0"
-browse: "https://${BASEURL}/pkg/"
-upstream: "https://github.com/ocaml/opam-repository/tree/master/"
+# Append to the 'repo' file, and dispatch all non-standard versions
+cat <<EOF >>repo
```

Comment:
```suggestion
sed -i 's/^browse: /d' repo
cat <<EOF >>repo
browse: "https://${BASEURL}/pkg/"
```

Reply:
but why? In opam-repository the `browse` points to `https://opam.ocaml.org/pkg` (and aside, what is `browse` used for?)

Reply:
> and aside, what is browse used for?

no idea. @rjbou do you know what it's used for?

Reply:
I removed my commit

Reply:
FWIW, https://opam.ocaml.org/pkg/ returns an empty page (404, Not found). In the opam sources, there's nothing using the `browse` from OpamFile.RepoSyntax. From the Manual.md: ``* <a id="repofield-browse">`browse: <string>`</a>:
  An URL where the users may browse available packages online
``